internal/service/credentials/resource_credential_type.go

```diff
@@ -161,18 +173,40 @@ func (r *CredentialTypeResource) Schema(ctx context.Context, req resource.Schema
 			),
 
 			"title": schema.StringAttribute{
-				Description: "Title of the credential. Verification sites are expected to be able to request the issued credential from the compatible wallet app using the title.",
-				Required:    true,
+				Description:         titleDescription.Description,
+				MarkdownDescription: titleDescription.MarkdownDescription,
+				Required:            true,
 				Validators: []validator.String{
 					stringvalidator.LengthAtLeast(attrMinLength),
+					customstringvalidator.IsRequiredIfRegexMatchesPathValue(
```

Comment:
This should also probably get a "bug" changelog entry, e.g.
https://github.com/pingidentity/terraform-provider-pingone/blob/35369cf5c82fe9601814c99994d1585e4aedddf7/.changelog/365.txt#L5-L7